ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -2096,19 +2102,13 @@ abstract class AbstractHttpServiceIntegrationTestTokenIndependent
       //The numContracts size should test for https://github.com/digital-asset/daml/issues/10339
       val numContracts: Long = 2000
       val helperId = domain.TemplateId(None, "Account", "Helper")
-      val payload = v.Record(
-        fields = List(v.RecordField("owner", Some(v.Value(v.Value.Sum.Party(alice.unwrap)))))
-      )
+      val payload = recordFromFields(ShRecord(owner = v.Value.Sum.Party(alice.unwrap)))
```

Comment:
We can, but this works a lot better if we integrate the containing record structure, and deal with

> Party lists are also a little inflexible, but maybe we should have fewer strings in this code.

You want to get as close as possible to something like this:

```scala
VA.record(ShRecord(owner = VA.party))
  .inj(ShRecord(owner = alice))
```

because the first line can be usefully factored and the second line is then type-checked all the way down. (There are already some examples of this in this file, but the record specification is too boilerplate-y in the current design.)

Reply:
Some improvement in #13619 but insufficient.